src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -10,3 +10,4 @@ CA2355 | Security | Disabled | DataSetDataTableInSerializableObjectGraphAnalyzer
 CA2356 | Security | Disabled | DataSetDataTableInWebSerializableObjectGraphAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2356)
 CA2361 | Security | Disabled | DoNotUseDataSetReadXml, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2361)
 CA2362 | Security | Disabled | DataSetDataTableInSerializableTypeAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2362)
+CA3077 | Security | Warning | DoNotUseInsecureDtdProcessingInApiDesignAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca3077)
```

Comment:
Because of the function we have missed this analyzer ID.

Reply:
@Evangelink I don't think we want this to be enabled as a warning by default. Either it should be disabled, or have default severity lesser then a warning.

Reply:
Fine by me, it was the old level so I kept it.